README.md

```diff
@@ -54,14 +55,26 @@ join_cohorts:
     cohort-joiner:[version]
       --lhs output/input_2021-*.csv
       --rhs output/input_ethnicity.csv
+      --output-dir output/joined
   needs: [generate_cohort, generate_ethnicity_cohort]
   outputs:
     highly_sensitive:
-      cohort: output/input_2021-*_joined.csv
+      cohort: output/joined/input_2021-*.csv
 ```
 
-For each left-hand cohort file, there will now be a corresponding joined file.
-For example, given a left-hand cohort file called `input_2021-01-01.csv`, there will now be a corresponding joined file called `input_2021-01-01_joined.csv`.
+Optionally, the following cohort-extractor action generates one or more measures from the joined cohorts:
```

Comment:
If we are going to update the documentation to describe expectations around output-dir and file regexes, would be good to link here. But in the meantime, might be helpful to note that the measures framework --output-dir has to be where the relevant input files are already located/ to note that generated measures files will be colocated with the input files used to produce them.